src/open_inwoner/scss/pdf/_jaaropgave.scss

```diff
@@ -15,6 +15,7 @@
 
   .yearly-report__p,
   .yearly-report__td {
+    font-family: utrecht-paragraph;
```

Comment:
`font-family: utrecht-paragraph;` will never work because "utrecht-paragraph" is an HTML CLASS. 
So this is not CSS. 
_If_ this would be an NLDS project file, then the correct variable would be `var(--utrecht-document-font-family)` which eventually cascades to the end-value 'Body' font-family, but it is not an NLDS page , so i will revert all of these values and add more for the headings.